sys-libs/ncurses/ncurses-6.3_p20220423.ebuild

```diff
@@ -99,7 +163,7 @@ src_configure() {
 		CXXFLAGS=${BUILD_CXXFLAGS} \
 		CPPFLAGS=${BUILD_CPPFLAGS} \
 		LDFLAGS="${BUILD_LDFLAGS} ${lbuildflags}" \
-		do_configure cross --without-shared --with-normal
+		do_configure cross --without-shared --with-normal --with-progs
```

Comment:
Does this install some extra stuff we didn't need before?

Reply:
No, ncurses 6.3 does not install additional binaries or tools under /usr/bin etc.
Both 6.2 and 6.3 install binaries like `clear` or `tic`.
I think `--with-progs` has been always the default.

Reply:
Ok, thanks for the explanation.